Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -557,11 +557,10 @@ def bottle_formula(f, args:)
           "date"     => Pathname(filename.to_s).mtime.strftime("%F"),
           "tags"     => {
             bottle_tag.to_s => {
-              "filename"              => filename.url_encode,
-              "local_filename"        => filename.to_s,
-              "sha256"                => sha256,
-              "formulae_brew_sh_path" => Homebrew::API::Formula.formula_api_path,
```

Comment:
I wouldn't mind a second set of eyes to confirm that this is safe to remove. Since https://github.com/Homebrew/brew/pull/12248, this has always returned `formula` and never `formula-linux`. There was only one reference to `formulae_brew_sh_path` in this repo [which I've replaced](https://github.com/Homebrew/brew/pull/14304/files#diff-4890b590cb90a2c9ef5af6c48b020b952b712a1b080fa39be1c377de8e8a7c62) with a hardcoded path since it no longer varies.

Reply:
That makes sense to me.

Reply:
Makes sense to me 👍🏻